kubernetes/bin/deploy.sh

```diff
@@ -1,53 +1,59 @@
 #!/bin/bash
 export DIR=kubernetes
 
-USAGE="USAGE: ./kubernetes/bin/deploy-development.sh create|create all|delete|delete all"
+USAGE="USAGE: ./kubernetes/bin/deploy.sh  api|staging create|create all|delete|delete all"
 
-if [[ "$1" = "create" ]];
+if [[ "$1" = "api" ]]
 then
+    export BRANCH=api
+else
+    export BRANCH=staging
+fi
+
+if [[ "$2" = "create" ]]
+then
+    echo "Starting '$BRANCH' branch deployment"
     echo "Creating objects from configurations."
     echo "Make sure that persistent disk is already created."
     echo ""
     echo "Creating Elasticsearch Deployment"
-        kubectl create -R -f ${DIR}/yamls/staging/elasticsearch/
+    kubectl create -R -f ${DIR}/yamls/${BRANCH}/elasticsearch/
     # For mqtt
-        kubectl create -R -f ${DIR}/yamls/mosquitto/
-    if [[ "$2" = "all" ]];
+    kubectl create -R -f ${DIR}/yamls/mosquitto/
+    if [[ "$3" = "all" ]];
     then
         # Start KubeLego deployment for TLS certificates
         kubectl create -R -f ${DIR}/yamls/lego/
         echo "Start nginx deployment, ingress & service"
         kubectl create -R -f ${DIR}/yamls/nginx/
     fi
     # Create web namespace
-    kubectl create -R -f ${DIR}/yamls/staging/web/
+    kubectl create -R -f ${DIR}/yamls/${BRANCH}/web/
     # Wait for some time to prevent any chance of API-Server to access incomplete deployments
     sleep 20
-    echo "Creating loklak deployment"
+    echo "Creating Loklak Server deployment"
     # Create API server deployment and service for development branch
-    kubectl create -R -f ${DIR}/yamls/staging/api-server/
-    echo "Deployed loklak on Kubernetes"
+    kubectl create -R -f ${DIR}/yamls/${BRANCH}/api-server/
```

Comment:
We would need to set some different name/namespace for staging and api deployments as they would conflict while updating images via Travis.